Description: fix desc.po problem and do not distclean tarball files.
 Backport upstream 1ca5ac8c5b commit.
Origin: upstream, http://code.google.com/p/fcitx/source/detail?r=1ca5ac8c5b8d921b777a09a52ddf855256fee3d3
Author: Aron Xu <happyaron.xu@gmail.com>
Last-Update: 2010-11-25
--- a/Makefile.am
+++ b/Makefile.am
@@ -11,3 +11,9 @@
         missing \
         mkinstalldirs \
         Makefile.in
+
+install-exec-local:
+	@echo
+	@echo Fcitx Install Finished.
+	@echo You may need to run ldconfig to make shared library work.
+	@echo
--- a/configure.in
+++ b/configure.in
@@ -39,12 +39,6 @@
 
 AC_SUBST(HOST_LIBS)
 
-AC_PATH_PROGS(WGET, wget, no)
-if test "x$WGET" = "xno";
-then
-    AC_MSG_ERROR([You need to install wget])
-fi
-
 AC_PATH_PROGS(TAR, tar, no)
 if test "x$TAR" = "xno";
 then
@@ -56,7 +50,6 @@
 then
     AC_MSG_ERROR([You need to install md5sum])
 fi
-AC_SUBST(WGET)
 AC_SUBST(TAR)
 AC_SUBST(MD5SUM)
 
@@ -120,7 +113,7 @@
             # Must use dbus, raise a error.
             AC_MSG_ERROR([No PANGOCAIRO found, configure for pango failed!])
         else
-            AC_MSG_WARN([No PANGOCAIR found, skipping configure for pango])
+            AC_MSG_WARN([No PANGOCAIRO found, skipping configure for pango])
         fi
   fi
 fi
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -16,7 +16,7 @@
 
 EXTRA_DIST = pySym.mb sp.dat punc.mb vk.conf gbks2t.tab AutoEng.dat gbkpy.org getdescpo fcitx-config $(configdescfile)
 CLEANFILES = $(py_data)
-DISTCLEANFILES = $(pyorg_data) $(py_tar)
+DISTCLEANFILES = $(pyorg_data)
 bin_SCRIPTS = fcitx-config
 
 @srcdir@/pybase.mb: @srcdir@/pyphrase.mb ;
@@ -31,16 +31,14 @@
 	cd $(srcdir) && $(TAR) xzvf $(py_tar)
 
 $(py_tar):
-	$(WGET) -O $(py_tar) $(py_url)
-	$(WGET) -O $(py_tar).md5 $(py_url).md5
 
 md5check: $(py_tar)
 	cd $(srcdir) && $(MD5SUM) -c $(py_tar).md5 || ( $(RM) $(py_tar); exit 1 )
 
-all-local: tools  $(py_data)
+all-local: tools  $(py_data) @srcdir@/desc.po
 	rm -f pyERROR
 	rm -f pyPhrase.ok
 
-desc.po: *.desc
-	./getdescpo
+@srcdir@/desc.po: @srcdir@/*.desc
+	@srcdir@/getdescpo @abs_srcdir@
 
--- a/data/getdescpo
+++ b/data/getdescpo
@@ -1,8 +1,10 @@
 #!/bin/sh
 filename=desc.po
-dir=`pwd`
+dir=$1
 filename=`basename $filename`
 
+cd $dir
+
 rm -f desc.po; touch desc.po
 
 cd ..
--- a/data/table/Makefile.am
+++ b/data/table/Makefile.am
@@ -39,7 +39,7 @@
 
 CLEANFILES = $(mb_data)
 
-DISTCLEANFILES = $(txt_data) $(table_tar)
+DISTCLEANFILES = $(txt_data)
 
 @srcdir@/%.mb: @srcdir@/%.txt
 	../../tools/txt2mb $< $@
